### PR TITLE
Improvised animals route

### LINKED ISSUE #132: Improvise animal routes with specialization
*Describe the feature*
Improve animal routes specializing them based on animal kingdom hierarchy as follows:
image

Like, currently add `animals/amphibians` and `animals/amphibians/:id` with the following key-value pairs:
`type`, `breed`, `properities`, `imageUrl`, `description`

*Tasks Todo*
- [ ] Add  json file with appropiate data in `data/animals/amphibianData.json`
- [ ] Add endpoint `animals/amphibians` in `animals.ts`
- [ ] Add enpoint `animals/amphibians/:id` in `animals.ts`

*Are you working on this feature? (Yes)*

### PR DESCRIPTION
**Pull request by ritik307**
# Related Issue

-   closes #132 

# Proposed Changes

- [x] Add  json file with appropiate data in `data/animals/amphibianData.json`
- [x] Add endpoint `animals/amphibians` in `animals.ts`
- [x] Add enpoint `animals/amphibians/:type` in `animals.ts`

# Checklist

-   [x] Tests
-   [x] Translations
-   [x] Documentation

### CODE REVIEW

**Comment by ritik307** [comment]
@paramsiddharth @ShauryaAg Plz review

**Comment by paramsiddharth** [comment]
@ritik307 Make it as follows:
1. GETting /animals returns all animals.
2. GETting /animals/amphibians returns amphibians.

As I can see, currently we only have amphibians (correct me if I am wrong). Once other categories are added, they should also be returned along with amphibians on GETting the /animals route.

File: data/animals/amphibiandata.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "data": [{
```
Comment (ShauryaAg) [inline data/animals/amphibiandata.json:2]: Put `{` in the next line.

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

**Comment by ritik307** [comment]
@ShauryaAg @paramsiddharth Plz review the changes.

**Review by ShauryaAg — APPROVED** [review]

**Review by paramsiddharth — APPROVED** [review]

**Comment by paramsiddharth** [comment]
@ritik307 Thank you!

**Pull request merged**